src/main/java/com/rultor/agents/github/Reports.java

```diff
@@ -106,7 +106,17 @@ public Reports(final Github ghub) {
             msg.append(Reports.tail(req));
         }
         final int number = Integer.parseInt(req.xpath("@id").get(0));
-        new Answer(Reports.origin(issue, number)).post(success, msg.toString());
+        final Comment.Smart comment = Reports.origin(issue, number);
+        final String message;
+        if (comment.body().contains("stop")) {
```

Comment:
@mkordas `QnStop` is used to create response like `OK, I'll try to stop current task.` But In current issue we deal with case when we return report, not answer on `stop` command.